### PR TITLE
Update subscription details in the readme.

### PR DESCRIPTION
**Pull request by dehora**
/ @valgog

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -662,21 +723,30 @@ HTTP/1.1 200 OK
 }
 ```
 
-#### Deleting Subscription
-To delete subscription one should run DELETE request on /subscriptions/{subscriptionId}
+#### Deleting a Subscription
+
+To delete a Subscription, send a DELETE request to the Subscription resource using its `id` field (`subscriptions/{id}`):
```
Comment (v-stepanov) [inline README.md]: I think leading slash is lost here, should be **/**subscriptions/{id}
(at least in other places we have a leading slash)

File: README.md [hunk 2]
```diff
@@ -662,21 +723,30 @@ HTTP/1.1 200 OK
...
+Successful response:
+
 ```
 HTTP/1.1 204 No Content
 ```
+
 #### Getting and Listing Subscriptions
-Getting single subscription by id:
+
+To view a Subscription send a GET request to the Subscription resource resource using its `id` field (`subscriptions/{id}`): :
```
Comment (v-stepanov) [inline README.md]: also leading slash is lost

**Comment by v-stepanov** [comment]
Everything else looks great 👍 
Thanks a lot for contribution, Bill. Now the Subscription documentation is much more better.

File: README.md (hunk 1 above)
Comment (valgog) [inline README.md]: @dehora @v-stepanov what is the progress here guys?, can we merge the branch and keep the PL open to follow up on these minor remarks?

**Comment by valgog** [comment]
(y)

**Comment by v-stepanov** [comment]
:shipit:

**Pull request merged**